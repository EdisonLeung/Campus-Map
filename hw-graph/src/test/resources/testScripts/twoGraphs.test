# Test the creation of 2 graphs. One graph will have 2 nodes with an edge, the other
# will only have 1 node

# Create graphs
CreateGraph graphOne
CreateGraph graphTwo

# Add 2 nodes to graphOne
AddNode graphOne a
AddNode graphOne b

# Add one node to graphTwo
AddNode graphTwo a

# Add edge for graphOne
AddEdge graphOne a b AB

# Print nodes and children of graphs
ListNodes graphOne
ListChildren graphOne a

ListNodes graphTwo