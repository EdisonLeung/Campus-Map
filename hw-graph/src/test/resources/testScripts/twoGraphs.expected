# Test the creation of 2 graphs. One graph will have 2 nodes with an edge, the other
# will only have 1 node

# Create graphs
created graph graphOne
created graph graphTwo

# Add 2 nodes to graphOne
added node a to graphOne
added node b to graphOne

# Add one node to graphTwo
added node a to graphTwo

# Add edge for graphOne
added edge AB from a to b in graphOne

# Print nodes and children of graphs
graphOne contains: a b
the children of a in graphOne are: b(AB)

graphTwo contains: a